### PR TITLE
MOTECH-2064 Updating MOTECH Roadmap doc

### PR DESCRIPTION
**Pull request by craigappl**
FYI @llavoie

### CODE REVIEW

File: docs/source/roadmap.rst [hunk 1]
```diff
@@ -1,9 +1,52 @@
...
+    - Improve UI build process with Node, Bower, Gulp and SCSS
+    - Split database from motech_data_services to motech_data and motech_schema
+
+
+Module Enhancements
+-------------------
+    - New IHE Interop Module supports the creation of HL7 CDA documents in MOTECH
+    - Improved OpenMRS module with tasks integration
+    - New Atom-Client module allows MOTECH to query atom feeds and act on changes in the feed. It will primarily be used in OpenMRS integration to detect when a new patient, encounter or provider has been created. However, this module can work with any atom feed.
+
```
Comment (llavoie) [inline docs/source/roadmap.rst]: Given MOTECH-2159, do we want to mention that we are releasing the atom feed in 0.28 or not? Do we need to know more about the bug before we decide what to say here?

**Comment by llavoie** [comment]
Looks mostly good -- I just have one question inline. Also looks like latest master needs to be merged in.

File: docs/source/roadmap.rst (hunk 1 above)
Comment (craigappl) [inline docs/source/roadmap.rst]: I verified this afternoon that the atom client module is working as intended and commented on the ticket my findings with a cc to you and Frank.

**Comment by craigappl** [comment]
merged master into the branch.

**Pull request merged**